### PR TITLE
Removed `go-sh` library.

### PR DESCRIPTION
**Pull request by EECOLOR**

### CODE REVIEW

**Comment by EECOLOR** [comment]
Not sure we need the second commit. I'll leave it up to you

**Comment by paulp** [comment]
I wrote it that way on purpose. It would have been easier to duplicate the cmd, args... signature, but I didn't want it. It makes it harder to compose to have one of the argument to exec hanging out on the side by itself. There's nothing special about the command name. They're all just arguments to the process invoker.

Think of any method you write which accepts a list which you expect to be non-empty. How often do you write it def `foo(firstArg T, otherArgs T...)`? Me, not very often.

File: suffuse/process.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package suffuse
 
 import (
-  "github.com/codeskyblue/go-sh"
```
Comment (paulp) [inline suffuse/process.go:4]: I do agree that this dependency is buying us zero. That looks to be a common theme with go dependencies, that they're often no improvement on nothing.

**Comment by EECOLOR** [comment]
I often write it like that actually, usually to let people know the first one is not optional.

But more importantly, you did this: `if len(args) == 0 {` which seemed like a waiste of characters. There is always one argument, it would not make sense otherwise. 

It felt like you tried to fight the command signature without gaining anything.

**Comment by paulp** [comment]
Well yeah, it wouldn't make sense otherwise, that's the point. You can fail with "index out of bounds" or you can issue an error. You can catch that error statically at the price of making it less convenient to use. I don't expect to ever catch that error statically ("it wouldn't make sense otherwise") so I don't want to make it less convenient to use.

**Comment by paulp** [comment]
If you think it's terribly important that the first argument be separated from all the other arguments then there should be a struct containing { commandName, args } which is passed around. That way there's only one entity and someone composing arguments from multiple sources isn't always stuck slicing the command name off of the list to please the exec method signature which requires them to be delivered in separately wrapped gift baskets.

That seems like unnecessary ceremony, but I'd prefer it to "x string, xs string...".

**Comment by EECOLOR** [comment]
I see what you mean.
1. Limit the change only to the `Command` object and keep the array and size check.
2. ... ok I will not

I'll stick with option one.

**Comment by EECOLOR** [comment]
Be aware of the force push.

**Pull request merged**